src/controller/CHIPDevice.h

```diff
@@ -383,6 +385,12 @@ class Device : public Messaging::ExchangeDelegate, public SessionEstablishmentDe
 
     void SetAddress(const Inet::IPAddress & deviceAddr) { mDeviceAddress.SetIPAddress(deviceAddr); }
 
+    void GetMRPIntervals(uint32_t & idleInterval, uint32_t & activeInterval)
```

Comment:
const?

Reply:
Do you mean `const uint32_t & idleInterval`? Why it should work for the reference?

Reply:
I think Łukasz means `const` method, at the end of the line.

Reply:
Oh ok, sorry I didn't get that :)